__tests__/split/oas3-no-errors/snapshot.js

```diff
@@ -0,0 +1,119 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`E2E split openapi with no errors 1`] = `
+
+type: object
```

Comment:
Why it prints definition into stdout? I don't undeerstand. Must be some mistake.

Reply:
If I place it after, components aren't splitter. The files are created, but refs are missing in the paths or webhooks. 
@RomanHotsiy I can investigate this.

Reply:
@RomanHotsiy the reason for the print is this condition - https://github.com/Redocly/openapi-cli/blob/master/packages/cli/src/utils.ts#L141
And we call `writeYaml` func several times while running the `split` command.
Should I delete this section? It might have been added for testing purposes. 
@ohorbachevskyi maybe you can recall?

Reply:
It's actually printing the content instead of creating the actual files. We can change it if needed or just leave the snapshot as-is.

Reply:
Okay, let it be then.